app/models/course_user.rb

```diff
@@ -31,6 +31,9 @@ def experience_points
     state :rejected
   end
 
+  scope :pending, -> { where(workflow_state: :pending) }
```

Comment:
The workflow gem provides the scope helper already, you don't need to declare here